cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerLoggingConfigurationDiff.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.operator.resource;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import io.fabric8.zjsonpatch.JsonDiff;
+import io.strimzi.operator.cluster.model.OrderedProperties;
+import io.strimzi.operator.common.operator.resource.AbstractResourceDiff;
+import org.apache.kafka.clients.admin.AlterConfigOp;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.ConfigEntry;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static io.fabric8.kubernetes.client.internal.PatchUtils.patchMapper;
+
+public class KafkaBrokerLoggingConfigurationDiff extends AbstractResourceDiff {
+
+    private static final Logger log = LogManager.getLogger(KafkaBrokerLoggingConfigurationDiff.class);
+    private static final ObjectMapper MAPPER = patchMapper().configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true);
```

Comment:
I think we probably don't want to force the rest of Fabric 8's patch code to use ordered maps when computing diffs. So we should probably instantiate our own `ObjectMapper` here since it's the only place where we do case about order (right?)